lib/src/importer-registry.ts

```diff
@@ -43,10 +53,31 @@ export class ImporterRegistry<sync extends 'sync' | 'async'> {
 
   /** Converts an importer to a proto without adding it to `this.importers`. */
   register(
-    importer: Importer<sync> | FileImporter<sync>
+    importer: Importer<sync> | FileImporter<sync> | NodePackageImporter,
+    _entryPointURL?: string | URL
   ): proto.InboundMessage_CompileRequest_Importer {
     const message = new proto.InboundMessage_CompileRequest_Importer();
-    if ('canonicalize' in importer) {
+    if (typeof importer === 'symbol') {
+      if (importer !== nodePackageImporter) {
+        throw 'Incorrect Node Package Importer used';
```

Comment:
This doesn't provide the user with much information about what was incorrect about it. I think it might make more sense to say `Unknown importer ${importer}`, since we don't know that they intended it to be a Node importer at all. Maybe a future version of Sass adds another symbol-based importer and they pass it to an older version of the API.

Reply:
Changed in b3f5414d3c366bed91a220089b2f201cec0c178a

Side note, unless the user provides a description for their Symbol, this will error with `Unknown importer Symbol()` since we won't have access to the importer name.